projects/wpdx/src/app/rehab-prio/rehab-prio.component.ts

```diff
@@ -841,6 +841,17 @@ export class RehabPrioComponent implements OnInit {
 
   } 
 
+  gotoPointFromId(id) {
+    const query = `select * from wpdx_enhanced where wpdx_id='${id}'`;
+    if (id !== undefined){
```

Comment:
better to ensure that id is not false-y and has correct length (i.e. that matches a wpdx_id).